src/Illuminate/Database/Eloquent/Relations/MorphTo.php

```diff
@@ -253,6 +263,24 @@ public function getDictionary()
         return $this->dictionary;
     }
 
+    /**
+     * Specify which relations to load for a given morph type.
+     *
+     * @param string $modelClass
+     * @param array $with
+     *
+     * @return \Illuminate\Database\Eloquent\Relations\MorphTo
+     */
+    public function withMorph(string $modelClass, array $with): self
```

Comment:
Laravel doesn't use return type-hints.

Reply:
Check. I copied over this implementation from one of our projects, I removed it now.